saleor-app.ts

```diff
@@ -40,6 +40,14 @@ switch (aplType) {
     throw new Error("Invalid APL config, ");
   }
 }
+
+if (!process.env.SECRET_KEY && process.env.NODE_ENV === "production") {
```

Comment:
its fine but it should be a part of app-sdk

Reply:
We do not _need_, but this code will fail at startup time instead of first metadata request

Reply:
Ok lets keep it